src/AssetTransferApi.ts

```diff
@@ -539,22 +555,32 @@ export class AssetTransferApi {
 	 * @param destChainId
 	 * @param specName
 	 */
-	private establishDirection(destChainId: string, specName: string, destIsSystemParachain: boolean): Direction {
+	private establishDirection(
+		isLocal: boolean,
```

Comment:
My thought would be to remove the `isLocal` check and keep that seperate, since `establishDirection` is really focusing on xcmDirections, and would help us from not having to typeCast. 

And when we need to check for the localTx we just do `if (isLocalTx) ...`

Reply:
That being said I am not deadset on removing `isLocal` happy to hear some counter thoughts.

Reply:
The reason I decided to add a `Local` `Direction` is because currently for txs that are `local balance transfers` on the relay chain, `establishDirection` returns the direction `RelayToPara`. There is no `RelayToRelay` and in the future this description won't necessarily be correct to treat as a `Local` relay chain transaction since there will be multiple Relay chains communicating with each other. So we could either allow the default for this case to continue to be `RelayToPara` so that `establishDirection` doesn't throw an error being unable to determine a direction or we could go with a solution similar to this. Open to discuss it more either way

Reply:
Ahh i see the reason, okay I think your implementation is sound than and totally works for me!